drivers/openstack/client.go

```diff
@@ -388,6 +389,22 @@ func (c *GenericClient) assignNeutronFloatingIP(d *Driver, floatingIP *FloatingI
 	return nil
 }
 
+func (c *GenericClient) DeleteFloatingIP(d *Driver, floatingIP *FloatingIP) error {
+	if d.ComputeNetwork {
+		// Not implemented
```

Comment:
Is that a FIXME, or is there something that prevents this from being implemented?

Reply:
IMHO the entire nova code could be removed as it is marked as [deprecated](https://docs.openstack.org/nova/pike/admin/networking-nova.html).

"New" features shouldn't care to implement this.

Reply:
Fair enough! I'd still add a warning to the tune of "Floating IP will not be removed, please do so manually if necessary" or something in those lines, but I agree that it's very low-impact.